publications-2018.bib

```diff
@@ -871,4 +871,29 @@ @article{sartori18
 	year = {2018}
 }
 
+
+@article{samrock2018,
+  title={Magnetotelluric image of transcrustal magmatic system beneath the {Tulu Moye} geothermal prospect in the {Ethiopian Rift}},
+  author={Samrock, Friedemann and Grayver, Alexander V and Eysteinsson, Hjalmar and Saar, Martin O},
+  journal={Geophysical Research Letters},
+  volume={45},
+  number={23},
+  pages={12--847},
```

Comment:
Page numbering incorrect. Website lists this as `Pages 12,847-12,855` , and I have no idea if the comma is a "thousands" delimiter or not.